editor/animation/animation_track_editor.h

```diff
@@ -577,7 +579,10 @@ class AnimationTrackEditGroup : public Control {
 	AnimationTrackEditor *editor = nullptr;
 
 	bool hovered = false;
+	bool track_edits_visible = true;
```

Comment:
Why does this need to exist if we already have this state in `Animation::collapsed_groups`?

Reply:
Good question; I liked the idea of keeping this sort of state information stored locally, but it is undeniably a level of data duplication. I'll look at removing it so that there's just one source of truth for this info.